build/package/circleci/orb.yml

```diff
@@ -95,16 +108,30 @@ jobs:
         description: "If provided, LaunchDarkly will attempt to generate links to your Git service provider per code reference. Example: `https://github.com/launchdarkly/ld-find-code-refs/blob/${sha}/${filePath}#L${lineNumber}`. Allowed template variables: `sha`, `filePath`, `lineNumber`. If `hunkUrlTemplate` is not provided, but `repoUrl` is provided, LaunchDarkly will automatically generate links for github or bitbucket repo types."
         type: string
         default: ""
-    executor:
-      name: default
-      proj_key: << parameters.proj_key >>
-      base_uri: << parameters.base_uri >>
-      context_lines: << parameters.context_lines >>
-      exclude: << parameters.exclude >>
-      repo_type: << parameters.repo_type >>
-      repo_url: << parameters.repo_url >>
-      default_branch: << parameters.default_branch >>
-      commit_url_template: << parmaeters.commit_url_template >>
-      hunk_url_template: << parameters.hunk_url_template >>
+      debug:
+        description: "Enables verbose debug logging."
+        type: boolean
+        default: false
+    docker:
+      - image: launchdarkly/ld-find-code-refs:0.5.0
     steps:
-      - find-flags
+      - checkout:
+          path: /repo
+      - run:
+          name: Find flag references
+          command: |
+            ld-find-code-refs \
+              -debug=<< parameters.debug >> \
+              -accessToken=<< parameters.access_token >> \
+              -projKey=<< parameters.proj_key >> \
+              -exclude=<< parameters.exclude >> \
+              -contextLines=<< parameters.context_lines >> \
+              -baseUri=<< parameters.base_uri >> \
+              -repoType=<< parameters.repo_type >> \
+              -repoUrl=<< parameters.repo_url >> \
+              -defaultBranch=<< parameters.default_branch >> \
+              -commitUrlTemplate=<< parameters.commit_url_template >> \
+              -hunkUrlTemplate=<< parameters.hunk_url_template >> \
```

Comment:
our new patch logic still works when `-hunkUrlTemplate=""` right? or do we asasume that the `-hunkUrlTemplate` is never provided at all.

Reply:
If the `hunk_url_template` parameter is not provided, `-hunkUrlTemplate` will resolve to an empty string, which is the same as the default. So the new(ish) logic around templates in `ld-find-code-refs` will still work as intended.